vars/getTestResults.groovy

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Maybe rename it getGradleTestResults? since these are specific to the gradle test?

Reply:
Renamed.